### PR TITLE
Create OOP-Geometry.py

### PR DESCRIPTION
**Pull request by ArtemDomnikoff**

### CODE REVIEW

File: 7-OOP-Geometry/OOP-Geometry.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import math
+
+
+class Circle:
+    def __init__(self, radius, result=0):
```
Comment (daria-gh) [inline 7-OOP-Geometry/OOP-Geometry.py]: Remove result 0

File: 7-OOP-Geometry/OOP-Geometry.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    def __init__(self, side, result=0):
+        self.side = side
+
+    def get_area(self):
+        self.result = self.side * self.side
+        return self.result
+
+
+circle_area = Circle(12)
+square_area = Square(12)
```
Comment (daria-gh) [inline 7-OOP-Geometry/OOP-Geometry.py]: Add another classes from issue

**Review by daria-larionova — APPROVED** [review]

File: 7-OOP-Geometry/OOP-Geometry.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        self.result = round(math.pi, 2) * self.radius * self.radius
+        return self.result
+
+
+class Square:
+    def __init__(self, side, result=0):
+        self.side = side
+
+    def get_area(self):
+        self.result = self.side * self.side
```
Comment (daria-gh) [inline 7-OOP-Geometry/OOP-Geometry.py]: why do we need self.result if we never use it? 
`return self.side * self.side`

File: 7-OOP-Geometry/Main.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+from Circle import *
```
Comment (daria-gh) [inline 7-OOP-Geometry/Main.py]: Preferable way is Python 3.
See the link https://stackoverflow.com/questions/4142151/how-to-import-the-class-within-the-same-directory-or-sub-directory

**Review by daria-larionova — CHANGES_REQUESTED** [review]

File: 7-OOP-Geometry/Circle.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import math
```
Comment (daria-gh) [inline 7-OOP-Geometry/Circle.py]: All the /__pycache__/*.pyc  files should be removed from branch

**Review by daria-larionova — CHANGES_REQUESTED** [review]

**Review by daria-larionova — APPROVED** [review]

**Pull request merged**